config/dba.neon

```diff
@@ -49,6 +50,7 @@ services:
                 - 'mysqli::execute_query#0'
                 - 'Doctrine\DBAL\Connection::query#0' # deprecated in doctrine
                 - 'Doctrine\DBAL\Connection::exec#0' # deprecated in doctrine
+                - 'Dibi\Connection::fetch#0'
```

Comment:
I guess since you added this method already in `SyntaxErrorInPreparedStatementMethodRule` we don't need it here again. you can verify this theses after you added test coverage and drop this line afterwards

Reply:
removed